kommunicate/src/main/java/com/applozic/mobicommons/people/channel/Channel.java

```diff
@@ -259,6 +260,10 @@ public int generateKmStatus(String loggedInUserId) {
             return CLOSED_CONVERSATIONS;
         }
 
+        if(getMetadata().containsKey(CONVERSATION_STATUS) && ("-1".equals(getMetadata().get(CONVERSATION_STATUS)))) {
+            return NOTSTARTED_CONVERSATIONS;
```

Comment:
@amntoppo how are we using this information to not showing initial conversation

Reply:
On the Agent app, it fetches the conversations according to the Conversation status of the channel: Assigned, Resolved and All.
Earlier, these initial conversations were kept as ALL_CONVERSATIONS and so this was also getting fetched when agent app fetches for all conversations. Now that we have created a separate status of these conversations, it will be fetched only when required, like when we search for them.